swiftnav_console/main.py

```diff
@@ -367,9 +369,11 @@ def _process_message_buffer(self, buffer):
                 data[Keys.LON_MIN] = m.solutionPositionStatus.lonMin
                 data[Keys.AVAILABLE_UNITS][:] = m.solutionPositionStatus.availableUnits
                 data[Keys.SOLUTION_LINE] = m.solutionPositionStatus.lineData
-                SolutionMap.send_pos(m.solutionPositionStatus)
+
+                if MAP_ENABLED:
+                    SolutionMap.send_pos(m.solutionPositionStatus)
```

Comment:
i dont really like this hacky business but:

the cpu / mem issue lies in sending position in the background
- this was done because sending an array of points via signals was more complex (i.e. sending array of pts then sending one by one updates, caching a bunch of pts etc...)
- but an issue is webengineview linked to the webchannel, (if theres a way to somehow decouple these classes? - sending points in the background should be fine as long as its not sending to webchannel)